src/animation/AnimationAction.js

```diff
@@ -21,7 +26,24 @@ THREE.AnimationAction = function ( clip, startTime, timeScale, weight, loop ) {
 	this.actionTime = - this.startTime;
 	this.clipTime = 0;
 
-	this.propertyBindings = [];
+	this.mixer = null;
+
+	var tracks = clip.tracks,
+		nTracks = tracks.length,
+		interpolants = new Array( nTracks );
```

Comment:
Not sure if you need (should) initalize an array with size. Probably better to just do `= []` (same below) 

see https://coderwall.com/p/h4xm0w/why-never-use-new-array-in-javascript for some thoughts ön this.